src/utils/BucketUtils.js

```diff
@@ -353,44 +353,42 @@ const createRpVoteStructureFromBuckets = (buckets) => {
 
 /**
  * Constrain the equality check to a limited number of fields. These
- * fields are the ones that are used in algorithm decisions and are what
+ * fields are the ones that are used in the v4 algorithm decisions and are what
  * determine whether it falls into a Data Use bucket. We limit the fields
- * because there are quite a few that have no impact on decision-making.
- * Fields like recontactMay and recontactMust, and others, are not relevant
- * to DAC decisions.
+ * because there are several captured values that have no impact on decision-making.
  *
  * @public
  * @param a Data Use
  * @param b Data Use
  * @returns {boolean}
  */
 export const isEqualDataUse = (a, b) => {
-  const fields = ['addiction',
-    'aggregateResearch',
-    'collaboratorRequired',
-    'controlSetOption',
-    'ethicsApprovalRequired',
-    'gender',
-    'geneticStudiesOnly',
-    'geographicalRestrictions',
-    'illegalBehavior',
-    'manualReview',
+  const fields = [
```

Comment:
Are we retiring `addiction`, `aggregateResearch`, `controlSetOption`, `manualReview`, and `otherRestrictions`?